adobe/spectrum-css
# refactor(assetlist)!: migrate to spectrum tokens

## Description

Migrates AssetList to spectrum tokens
- addresses empty label a11y issue (on docs site, storybook, and within Miller Column)
- Adds `is-navigated` AssetList-Item to docs site 
- Adds WCHM values 
- Addresses previous issue with text color on dark and darkest and focus indicators 



## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.



### Test Outline: 
@jawinn 
1. Open the [docs site](https://pr-2211--spectrum-css.netlify.app/assetlist) for the asset list component: 
- [x]  there is a 4th asset item, Downloads with `is-navigated` styling 
- [x] Tabbing through there is a focus indicator 
- [x] component contrast is appropriate for all color themes (light, dark, darkest) 
- [x] a11y testing has an accessible name for each check box 

2. Open the [storybook](https://pr-2211--spectrum-css.netlify.app/preview/?path=/docs/components-asset-list--docs) for the asset list component:
- [x] there are no a11y violations 
- [x] Hover, selected, and navigated all match docs site


### Regression testing

Validate:

1. A legacy documentation page (such as [accordion](https://pr-###--spectrum-css.netlify.app/accordion.html)), including:


4. A migrated documentation page (such as [action group](https://pr-###--spectrum-css.netlify.app/actiongroup.html)), including:


## Screenshots



## To-do list
